packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -125,7 +125,11 @@ class ModalLauncherKeypressListener extends React.Component<{
     }
 
     _handleKeyup = (e: KeyboardEvent) => {
-        if (e.key === "Escape") {
+        const escape = 27;
```

Comment:
I like this.

Reply:
Changed back to use `e.key`. `which` and `keyCode` are obsolete and/or non-standard, and their replacement `code` is a string. Pretty sure key is what we want.

Reply:
I've added a comment accordingly.